### PR TITLE
Upgrade operator-sdk to 1.1

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
I've discovered that we still depend on operator-sdk v0.17.0.
So, as far as I understand earlier we merged PR that did most of upgrade to OperatoSDK 1.1 which is moving to controller-gen.
This PR upgrades operator-sdk itself to 1.1. Let me know if I missed something and we should not do this upgrade.

Apart from that, it contains some minor improvements as self-descriptive commits.

### What issues does this PR fix or reference?
https://github.com/devfile/devworkspace-operator/issues/180

### Is it tested? How?
I've checked that cloud-shell with openshift-oauth routings works fine on crc 1.18.0(OpenShift 4.6.1 inside).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -124,11 +127,11 @@ debug: _print_vars _generate_related_images_env
 	WATCH_NAMESPACE=$(NAMESPACE) dlv debug --listen=:2345 --headless=true --api-version=2 ./main.go --
 
 ### install: Install CRDs into a cluster
-install: manifests _kustomize
+install_crds: manifests _kustomize _init_devworkspace_crds
```
Comment (amisevsk) [inline Makefile:130]: Need to also update doc line above (`### install`)

**Review by amisevsk — APPROVED** [review]
LGTM, good catch.

**Review by JPinkney — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (sleshchenko) [reply Makefile:130]: but then where will be confusion point :smile: 

Good catch. Will fix

**Pull request merged**